packages/polaris-viz/src/components/Grid/components/GroupInfo.tsx

```diff
@@ -53,17 +51,18 @@ export const GroupInfo: React.FC<GroupInfoProps> = ({
     [characterWidths],
   );
 
-  const divider = showNameAndSecondaryValue
-    ? PRIMARY_VALUE_WIDTH_RATIO
-    : PRIMARY_VALUE_WIDTH_RATIO_SOLO;
+  const valueAndSecondaryValue = showNameAndSecondaryValue
+    ? `${groupValue}${VALUES_DIVIDER}${groupSecondaryValue}`
+    : groupValue;
+  const truncatedValueAndSecondaryValue = getTruncatedText(
+    valueAndSecondaryValue,
+    groupWidth,
+  );
+  const truncatedValue =
+    truncatedValueAndSecondaryValue.split(VALUES_DIVIDER)[0];
+  const truncatedSecondaryValue =
+    truncatedValueAndSecondaryValue.split(VALUES_DIVIDER)[1];
```

Comment:
sure! the idea is to check against the full text combining the `groupValue` and `groupSecondaryValue` and since they are in separate `tspan` elements if we don't combine them the truncation works separately for the `value` and `secondaryValue`